### PR TITLE
Basic Elevator Control w/ Encoders

### PR DESCRIPTION
**Pull request by Jkvelagapudi**

### CODE REVIEW

**Comment by DragonflyRobotics** [comment]
Are there more commits left on this PR or would you like us to formally review the code?

**Comment by Jkvelagapudi** [comment]
You can review the code

**Comment by DragonflyRobotics** [comment]
This code looks wonderful and well-formatted. Great work!!. I am approving the merge but will hold off until we have the elevator assembly since I don't want to risk breaking the drivetrain yet.

File: shuffleboard-layout.json [hunk 1]
```diff
@@ -305,6 +305,19 @@
               "Visuals/Show tick mark ring": true,
               "Visuals/Counter clockwise": false
             }
+          },  
+          "4,3": {
```
Comment (DragonflyRobotics) [inline shuffleboard-layout.json:309]: Looks like just a text entry to Shuffleboard which is all well and good

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -35,7 +35,10 @@ public static class ModuleConstants {
 
   }
 
-  public static class ManipulatorConstants {}
+  public static class ManipulatorConstants {
+    public static final int elevatorMaster = 51;
```
Comment (DragonflyRobotics) [inline src/main/java/frc/robot/Constants.java]: Good naming and placement

File: src/main/java/frc/robot/RobotContainer.java [hunk 3]
```diff
@@ -173,20 +175,36 @@ public static double getManipulatorRightJoyX() {
...
   }
 
-  public static double getManipulatorLeftTrigger() {
-    if (Math.abs(m_manipulatorController.getLeftTriggerAxis()) > Constants.OperatorConstants.joystickDeadband) {
-      return m_manipulatorController.getLeftTriggerAxis();
-    } else {
-      return 0;
-    }
+  public static double getElevatorRightTrigger() {
+    if (Math.abs(m_driverController.getRightTriggerAxis()) > Constants.OperatorConstants.joystickDeadband) {
```
Comment (DragonflyRobotics) [inline src/main/java/frc/robot/RobotContainer.java:213]: Using deadbands which is great!

File: src/main/java/frc/robot/subsystems/Elevator.java [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  private final CANSparkMax elevatorMotor2;
+
+  public final RelativeEncoder elevatorEncoder;
+
+
+  public Elevator() {
+    elevatorMotor1 = new CANSparkMax(Constants.ManipulatorConstants.elevatorMaster, MotorType.kBrushless);
+    elevatorMotor2 = new CANSparkMax(Constants.ManipulatorConstants.elevatorFollower, MotorType.kBrushless);
+
+    elevatorMotor2.setInverted(false);
```
Comment (DragonflyRobotics) [inline src/main/java/frc/robot/subsystems/Elevator.java:29]: Correctly inverted motor!

**Review by DragonflyRobotics — APPROVED** [review]
I have left all good comments for other reviewers to see but I can certainly approve these changes!

**Pull request merged**